src/main/java/shikabot/task/TaskList.java

```diff
@@ -0,0 +1,54 @@
+package shikabot.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    public ArrayList<Task> taskList;
```

Comment:
Perhaps this should be protected or private. It would be better to use getters and setters to gain access to the variable